hrui/js/contact.js

```diff
@@ -0,0 +1,12 @@
+(function ($, _) {
+  $(document).on('crmLoad', function(e) {
+    $('.crm-inline-edit').one('DOMSubtreeModified', function() {
+      if ($(this).find('form').length === 1) {
+        $(this).find('form label').each(function() {
```

Comment:
Try to store jquery values in a variable, instead of querying multiple times. DOM operations are "expensive".